test/tests/mesh/add_subdomain_ids/add_subdomain_names.i

```diff
@@ -0,0 +1,52 @@
+[Mesh]
+  [generate]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 10
+    ny = 10
+  []
+  add_subdomain_names = 'MyAdditionalSubdomain'
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+  [diff2]
+    type = MatDiffusion
+    diffusivity = 2
+    variable = u
+    block = 'MyAdditionalSubdomain'
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  []
+  [right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  []
+[]
```

Comment:
```suggestion
```

this is an antiquated way of creating mesh tests.
We now use `cli_args = '--mesh-only'` to create tests that only check the mesh